cmd/createtree/main.go

```diff
@@ -81,18 +81,41 @@ func createTree(ctx context.Context) (*trillian.Tree, error) {
 	defer conn.Close()
 
 	client := trillian.NewTrillianAdminClient(conn)
+	var tree *trillian.Tree
 	for {
-		tree, err := client.CreateTree(ctx, req)
-		if err == nil {
-			return tree, nil
+		glog.Info("CreateTree...")
+		tree, err = client.CreateTree(ctx, req)
+		if err != nil {
+			if s, ok := status.FromError(err); ok && s.Code() == codes.Unavailable {
```

Comment:
Should this error be propagated back to the user rather than retrying internally?